.github/workflows/tests.yaml

```diff
@@ -28,11 +28,11 @@ jobs:
       run: docker build -t civiform-dev --cache-from docker.io/civiform/civiform-dev:latest ./
     - name: Run tests
       run: USE_LOCAL_CIVIFORM=true bin/run-test-ci
-    - name: jacoco-badge-generator
-      uses: cicirello/jacoco-badge-generator@v2
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v2
```

Comment:
I believe there was discussion around not using the CodeCov tool. @goldblatt should have more context. Can you confirm that there aren't any concerns there?

I'll review the rest of this PR as if that's already been figured out.

Reply:
She said yes to using Codecov in the last meeting. I will check with her once more.

Reply:
She said yes to using Codecov in the last meeting. I will check with her once more.

Reply:
cool ya i think we are okay the fear would be another attack on codecov that would allow the secrets in this particular file to be read. @swatkat1 will you add checking https://about.codecov.io/security-update/ to the oncall rotaiton docs please?